#pragma once

namespace NYT::NYTree {

////////////////////////////////////////////////////////////////////////////////

enum class ESerializableArcadiaEnum
{
    First,
    Second,
    Third,
};

////////////////////////////////////////////////////////////////////////////////

} // namespace NYT::NYTree
